### ISSUE TITLE
In editor release notes doesn't open external stylesheet

### ISSUE THREAD

**Issue by svenvNL** [Microsoft/vscode#22668]
- VSCode Version: Code 1.10.2 (8076a19fdcab7e1fc1707952d652f0bb6c6db331, 2017-03-08T14:02:52.799Z)
- OS Version: Windows_NT ia32 10.0.14393
- Extensions: 

|Extension|Author|Version|
|---|---|---|
|gitignore|codezombiech|0.5.0|
|vscode-eslint|dbaeumer|1.2.7|
|php-debug|felixfbecker|1.10.0|
|php-intellisense|felixfbecker|1.1.3|
|php-pack|felixfbecker|1.0.1|
|file-icons|file-icons|1.0.3|
|vscode-apache|mrmlnc|1.1.1|
|cpptools|ms-vscode|0.10.3|
|Theme-TomorrowKit|ms-vscode|0.1.3|
|java|redhat|0.0.11|
|stylelint|shinnn|0.23.0|
|arduino-vscode|steveyin|0.2.1|
|Seti-theme|TsumiNa|0.0.1|
|babelrc|waderyan|1.0.0|
|twig|whatwedo|1.0.2|;

---

Steps to Reproduce:

1. Open the DevTools
2. Open the release notes (Help > Release Notes)
2. In the console, the following message will appear: 
```
[Embedded Page] Refused to load the stylesheet 'https://code.visualstudio.com/raw/css/inproduct_releasenotes.css' because it violates the following Content Security Policy directive: "style-src 'nonce-1489583323330330'".
```

**Comment by joaomoreno** [comment]
This is as expected.

**Comment by svenvNL** [comment]
@joaomoreno as I understand it, the Release Notes view loads the release notes as markdown from code.visualstudio.com. In the markdown there is a link to "inproduct_releasenotes.css" which is deleted for use on the main site, but kept in the raw markdown that Visual Studio Code uses.

If I remember correctly, there used to be a sort-of pink coloured floating action button that would scroll the editor to the top, but I don't see that button, alto the CSS looks related to that button.

Why is it excepted to try to load that stylesheet and block the request and block it by security rules? Why not remove the link element from the markdown source or loosen the security rule or ship the CSS in the product itself?

**Comment by joaomoreno** [comment]
OH, totally right, this is not as expected!

**Comment by joaomoreno** [comment]
@mattbierner Did the security fix break this?

**Comment by mjbvz** [comment]
@joaomoreno Yes, but I checked in a fix about two weeks back: 2337a9817b4d689b6038fb9e1871d4612a9f25a4

Let me know if you still see this in the latest insiders

**Comment by svenvNL** [comment]
@mjbvz I can the fab again and the dev tools don't show the message so the bug is fixes for me in insider.
